gui/wxpython/timeline/g.gui.timeline.py

```diff
@@ -24,7 +24,7 @@
 # % keyword: general
 # % keyword: GUI
 # % keyword: temporal
-# % keywords: plot
+# % keyword: plot
```

Comment:
What should be always used now? Here you normalized two files to "keyword:", but down below changed the pattern extracting for "keywords:"

Reply:
It is different place. Plural is a list of keywords in YAML front matter in Markdown. Singular is one keyword in interface definition which eventually feeds that list but is defined keyword by keyword.

Reply:
Good!